docs/_toc.yml

```diff
@@ -145,3 +145,6 @@ parts:
 
       # Doc contribution
       - file: CONTRIBUTING.md
+
+      # Prometheus / Alerting
+      - file: content/prometheus/README.md
```

Comment:
can we maybe position this elsewhere? I prefer to have the contribution/templates at the bottom. 

let's put it next to grafana.